README.md

```diff
@@ -1,98 +1,187 @@
+    - Using an IRC client and joining the [#qgis](http://webchat.freenode.net/?channels=#qgis) channel on irc.freenode.net.
+    - Using a Matrix client and joining the [#qgis:matrix.org](http://matrix.to/#/#qgis:matrix.org) room.
+    - Using [Gitter](https://gitter.im/qgis/QGIS?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) chat.
+ - At the [GIS stackexchange](https://gis.stackexchange.com/) or [r/QGIS reddit](https://www.reddit.com/r/QGIS/), which are not maintained by the QGIS team, but where the QGIS and broader GIS community provides lots of advice
+- [Other support channels](https://qgis.org/en/site/forusers/support.html)
 
-## Support
-You can get support in the following ways:
+## Get involved with the community
 
- -  Using the QGIS community site at https://qgis.org
- -  Joining the [qgis-users mailing list](https://lists.osgeo.org/mailman/listinfo/qgis-user)
- -  Chatting with us real-time.
-    Please wait around for a response to your question as many folks
-    on the channel are doing other things and it may take a while for
-    them to notice your question.
-    The following paths all take you to the same chat room:
-     - Using an IRC client and joining the [#qgis](http://webchat.freenode.net/?channels=#qgis) channel on irc.freenode.net.
-     - Using a Matrix client and joining the [#qgis:matrix.org](http://matrix.to/#/#qgis:matrix.org) room.
-     - Using [Gitter](https://gitter.im/qgis/QGIS?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) chat.
+### Bug reporting and bug fixing
 
-## Contribute
+You can help us **by submitting bug reports or fixing bugs** in the [QGIS bug tracker](https://github.com/qgis/QGIS/issues/).
 
-QGIS is on GitHub at https://github.com/qgis/QGIS. If you wish to contribute
-patches you can [fork the project](https://help.github.com/forking/), make your changes, commit to your
-repository, and then [create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/). The development team can then review your contribution and commit it upstream as appropriate.
+### New features and enhancements
+
+If you wish to contribute patches you can [fork the project](https://help.github.com/forking/), make your changes, commit to your repository, and then [create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/). The development team can then review your contribution and commit it upstream as appropriate.
 
 If you commit a new feature, add [FEATURE] to your commit message AND give a clear description of the new feature. A webhook will automatically create an issue on the QGIS-Documentation repo to tell people to write documentation about it.
```

Comment:
Sorry, not following exactly what change is needed here. I'll invite you as contributor to my branch/fork and you can adjust this (and other things) directly there, if you wish, or post-merge in master of course.

Reply:
I did not make a suggestion because I failed to find the right balance and wondered if it should be removed. Here's a suggestion anyway.
```suggestion
If you commit a new feature, add `[FEATURE]` to your commit message AND give a clear description of the new feature. The label `Needs documentation` will be added by maintainers and will automatically create an issue on the QGIS-Documentation repo to tell people to write documentation about it.
```
But I don't like the "tell people to write documentation" part since it may convey the message that the developer is not welcome to doc his work himself (there's already people to do that).

Reply:
Ah, now I understand. I just put in the following, based on your suggestion but inviting documentation by the contributor:

> The label `Needs documentation` will be added by maintainers and will automatically create an issue on the QGIS-Documentation repo, where you or others should write documentation about it.

Reply:
Nicer